app/views/govuk_template.scala.html

```diff
@@ -47,7 +47,9 @@
             @if(!CookiePolicy.bannerSeen(request)) {
                 <div id="global-cookie-message" class="cookie-banner">
                     <div class="content-container" role="alert">
-                        <span class="copy-16">GOV.UK uses cookies to make the site simpler. <a href="">Find out more about cookies</a></span>
+                        <span class="copy-16">GOV.UK uses cookies to make the site simpler.
+                            <a href="@routes.ApplicationController.privacy">Find out more about cookies</a>
```

Comment:
should the call to privacy have parenthesis, if we are to be consistent the the change ApplicationController:35?